src/index.html

```diff
@@ -5,8 +5,425 @@
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="stars">
+          <div class="stars__stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="stars__reviews">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="price">
+          <div class="price__card">
+            Price:
+          </div>
+          <div class="price__numbers">
+            $2,199
+          </div>
+        </div>
+
+        <a href="#">
+          <button class="button" type="button">buy</button>
+        </a>
+      </block>
+      <block class="card">
+        <img
+        src="./images/imac.jpeg"
+          class="card__image"
+          alt="imac"
+        >
+
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="stars">
+          <div class="stars__stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="stars__reviews">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="price">
+          <div class="price__card">
+            Price:
+          </div>
+          <div class="price__numbers">
+            $2,199
+          </div>
+        </div>
+
+        <a data-qa="card-hover" href="#">
+          <button class="button" type="button">buy</button>
+        </a>
+      </block>
+      <block class="card">
+        <img
+        src="./images/imac.jpeg"
+          class="card__image"
+          alt="imac"
+        >
+
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="stars">
+          <div class="stars__stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="stars__reviews">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="price">
+          <div class="price__card">
+            Price:
+          </div>
+          <div class="price__numbers">
+            $2,199
+          </div>
+        </div>
+
+        <a href="#">
+          <button class="button" type="button">buy</button>
+        </a>
+      </block>
+      <block class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```